Hazelcast.Test/Hazelcast.Client.Test/HazelcastTestSupport.cs

```diff
@@ -39,8 +39,13 @@ public class HazelcastTestSupport
 
         public HazelcastTestSupport()
         {
-            Environment.SetEnvironmentVariable("hazelcast.logging.level", "finest");
+#if DEBUG
```

Comment:
was this for local testing purposes? do we have DEBUG set all the time?

Reply:
This is removed on non Debug compiles. We only use Debug on local development, all ther case are not DEBUG.